tests/PINCacheTests/PINCacheTests.m

```diff
@@ -33,6 +34,9 @@ - (void)setUp
 - (void)tearDown
 {
     [self.cache removeAllObjects];
+    
+    // Disk cache needs a bit more time to clean up its trash 
+    sleep(1);
```

Comment:
I wonder if instead, we should make the disk cache unique for every test? I.e. in the setup create a cache with an NSUUID in the name?